config.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"math/rand"
+	"os"
+	"time"
+)
+
+var host string
+var jwtKey string
+
+func initConfig() {
+	host = os.Getenv("LN_SERVER_BASE_URL")
+	jwtKey = os.Getenv("LN_JWT_KEY")
+
+	if host == "" {
+		host = "people.sphinx.chat"
```

Comment:
is there a way to check what URL the go instance it is hosted on? it might make more sense to do that rather than hard code to `people.sphinx.chat` but we can do that later

Reply:
I didn't find a good way to get the server domain name with Golang,  os.Hostname() only gets the host name of the computer